rackcity/api/serializers/it_instance_serializers.py

```diff
@@ -9,6 +10,11 @@ class ITInstanceSerializer(serializers.ModelSerializer):
     Serializes all fields on ITInstance model, where model and rack fields are
     defined by their pk only.
     """
+    hostname = serializers.CharField(validators=[
+        UniqueValidator(
+            queryset=ITInstance.objects.all(), lookup='iexact'
+        )])
+
```

Comment:
case insensitive uniqueness will get validated whenever serializer.is_valid is called on this serializer